cmd/restic/integration_test.go

```diff
@@ -1283,6 +1283,67 @@ func TestPruneWithDamagedRepository(t *testing.T) {
 	t.Log(err)
 }
 
+// Test repos for edge cases
+func TestEdgeCaseRepos(t *testing.T) {
+	opts := CheckOptions{}
+
+	// repo where index is completely missing
+	// => check and prune should fail
+	testEdgeCaseRepo(t, "repo-index-missing.tar.gz", opts, false, false)
```

Comment:
Please use `t.Run(...)` to make these subtests selectable. That could come in handy when hunting a specific problem.

Reply:
Thanks for the hint about how to make subtests - I wasn't aware of this.
Is changed now.